frame/nomination-pools/src/lib.rs

```diff
@@ -963,9 +973,13 @@ impl<T: Config> BondedPool<T> {
 	}
 
 	/// Issue points to [`Self`] for `new_funds`.
+	/// Increase the TVL by the amount of funds.
```

Comment:
Please reference code using the [`...`] syntax and actual variables names where possible

```suggestion
	/// - Issues points to [`Self`] for `new_funds`.
	/// - Increases the [`TotalValueLocked`] by `new_funds`.
```